capabilities/promtail/zarf.yaml

```diff
@@ -1,11 +1,27 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/defenseunicorns/zarf/main/zarf.schema.json
```

Comment:
we don't put these tags in zarf.yaml anymore as they are loaded via the project root config in vscode

Reply:
no problem i already had them there i can remove